From: "Barak A. Pearlmutter" <barak+git@cs.nuim.ie>
Date: Wed, 19 Oct 2011 23:21:53 +0100
Subject: install manpages

Comment out code to install man pages in *.UTF-8/man*/.
---
 install-manpages |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/install-manpages b/install-manpages
index 6ea951c..7b6010e 100755
--- a/install-manpages
+++ b/install-manpages
@@ -15,8 +15,8 @@ for lingua in ${LINGUAS} ; do
 		continue
 	fi
 
-	${INSTALL} -d "${mandir}/${lingua}.UTF-8/man1"
-	${INSTALL} -m 644 "man/${lingua}/dwdiff.1" "${mandir}/${lingua}.UTF-8/man1"
+#	${INSTALL} -d "${mandir}/${lingua}.UTF-8/man1"
+#	${INSTALL} -m 644 "man/${lingua}/dwdiff.1" "${mandir}/${lingua}.UTF-8/man1"
 
 	${INSTALL} -d "${mandir}/${lingua}/man1"
 	${INSTALL} -m 644 "man/${lingua}/dwdiff.1" "${mandir}/${lingua}/man1"
-- 
